src/server/api/routers/blogPost.ts

```diff
@@ -47,6 +51,16 @@ export const blogPostRouter = createTRPCRouter({
         nextCursor,
       };
     }),
+  count: publicProcedure
+      .input(BlogPostWhereInputObjectSchema)
+      .query(async ({ input, ctx }) => {
+          const count: number = await ctx.prisma.blogPost.count({
+              where: {
+                  userId: input.userId,
```

Comment:
Since the input can take any "where" condition, instead of extracting the userId, doing:

```
where: input
```

would enable the usage of this API to the users that the signature promises. (The input type takes any 'where' condition, if the user passes anything else than userId, this API wouldn't function as promised)

If you want limited consumption (limited to userId), than change the input type to:
```
{
    userId: string,
}
```

Reply:
done